packages/core/admin/shared/contracts/shared.ts

```diff
@@ -0,0 +1,43 @@
+import type { Entity } from '@strapi/types';
+
+export type Permission = {
```

Comment:
do ignore the type definitions of these ones, they have to be refined a bit. They are only here to start working on the contracts itself.